Exploration Queries/InputEntity_Process/Process2Host_VMConfigChange.txt

```diff
@@ -0,0 +1,46 @@
+// Id: e32a48a9-bf82-4cec-ba94-9ec406a69ef8
+// Name: 10 most recent VM configuration changes based on process
+// Description: Provides the 10 most recent changes to any VM based on the process name provided
+// InputEntityType: Process
+// InputFields: [Process, ServiceFileName]
+// OutputEntityTypes: [Host]
+// QueryPeriodBefore: 1d
+// QueryPeriodAfter: 1d
+// DataSource: ConfigurationChange
+// Tactics: #Persistence, #Execution
+
+let ConfigChange = (v_Process_ImageFile_Name:string, v_Service_FileName:string,  ) {ConfigurationChange | where TimeGenerated >= ago(1d)
```

Comment:
Fixed these up, have a look.  We may need to chat about Alert V3 Schema...there is no indication of what to use for Service, so I used OneCyber.